config.h.cmake

```diff
@@ -1,3 +1,21 @@
+# This file is part of the dune-stuff project:
+#   https://github.com/wwu-numerik/dune-stuff
+# The copyright lies with the authors of this file (see below).
+# License: BSD 2-Clause License (http://opensource.org/licenses/BSD-2-Clause)
+# Authors:
+#   Felix Schindler (2013 - 2015)
+#   Rene Milk       (2013 - 2015)
+#   Tobias Leibner  (2014)
+
+// This file is part of the dune-stuff project:
+//   https://github.com/wwu-numerik/dune-stuff
+// The copyright lies with the authors of this file (see below).
+// License: BSD 2-Clause License (http://opensource.org/licenses/BSD-2-Clause)
+// Authors:
+//   Felix Schindler (2013 - 2015)
+//   Rene Milk       (2013 - 2015)
+//   Tobias Leibner  (2014)
+
```

Comment:
this one seems to be contained in both config files, should only be in `other` (ignore in `cpp`)